.github/ISSUE_TEMPLATE/00-use-case.md

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+---
+name: Use case
+about: Describe a new use case for the specification.
+title: ''
+labels: 'use case :bulb:'
+assignees: ''
+
+---
+
+**Description**
+
+As a:
+
+- [ ] Application User (application developer)
```

Comment:
"application user" and "application developer" are two very different personas. Did you perhaps mean "API User (Application Developer)"?

Reply:
done